app/views/assignments/create_lti_line_items.erb

```diff
@@ -0,0 +1,36 @@
+<div class="table lti-table">
+  <table>
+    <thead>
+    <tr>
+      <th><%= t("lti.lti_settings") %></th>
+      <th><%= t("lti.create_line_item_checkbox")%></th>
```

Comment:
In terms of style, each `%>` should have exactly one preceding space. In addition to this line, there are a couple in the file below that have 2 spaces.